src/components/pages/vision/introduction.tsx

```diff
@@ -52,20 +67,42 @@ const Introduction: React.FC = () => {
           Vision
         </h1>
       </div>
-      <div className="w-full mx-auto md:max-w-md">
+      <div className="w-full ">
         {sectionCopy.map((copy, i) => {
           return (
-            <div key={i} className="mt-20 sm:mt-30">
-              <h2 className="mb-10 text-3xl font-bold sm:text-4xl sm:mb-14">
-                {copy.title}
-              </h2>
-              {copy.descriptions.map((description, index) => {
-                return (
-                  <p className="mt-8" key={index}>
-                    {description}
-                  </p>
-                )
-              })}
+            <div
+              key={i}
+              className="grid grid-cols-1 gap-8 mt-20 sm:mt-40 sm:gap-20 md:grid-cols-2"
+            >
+              <div className={i === 1 ? 'sm:order-last' : ''}>
```

Comment:
Probably could use a `first:` pseudo-class-variant here instead of using JS

https://tailwindcss.com/docs/pseudo-class-variants

Reply:
Was able to use even: for this part vs js in the classname.